packages/classic-webview/src/pages/PlayPage.tsx

```diff
@@ -123,79 +123,80 @@ export const PlayPage = () => {
   );
 
   return (
-    <PageContentContainer showContainer={isHardcore}>
-      <div className="flex h-full flex-col items-center justify-center p-6">
```

Comment:
I deleted the `div` and applied its styles directly to the `PageContentContainer`. There were no more changes made in this file.